person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1.100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    of_age || @parent_permission
```

Comment:
- [ ] You have made a syntax error here that gives an **'undefined local variable or method'** error for of_age. You defined `of_age?` but you are calling `of_age` (without the question mark). Kindly fix this